### DISCUSSION TITLE
verification of newest versions of xz utils are nt affected by backdoor CVE-2024-3094

### DISCUSSION THREAD

**Discussion by zebulak** [git-for-windows/git#4892] [Q&A] [answered]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

** insert your machine's response here **
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

** insert your machine's response here **
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

** insert your machine's response here **
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** insert your response here **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** insert your response here **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** insert your commands here **
```
 - What did you expect to occur after running these commands?

** insert here **

 - What actually happened instead?

** insert here **

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** insert URL here **

**Comment by dscho** [comment, marked as answer]
Neither is `sshd` started by Git for Windows by default, nor is it based on `glibc` on Windows, nor are any Debian or RedHat type of packages involved. So no, Git for Windows (and the `sshd.exe` that is for historical reasons included in it) are affected.
